tests/e2e/normalize.go

```diff
@@ -208,6 +208,7 @@ func normalizeKRMObject(t *testing.T, u *unstructured.Unstructured, project test
 
 	// Specific to EssentialContactContact
 	visitor.replacePaths[".validateTime"] = "2024-04-01T12:34:56.123456Z"
+	visitor.replacePaths[".status.externalRef"] = "projects/${projectNumber}/contacts/${contactId}"
```

Comment:
I don't think this works. It will replace the `externalRef` in all tests. We need to parse the string. Something similar to https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/651cab365912dcc94f516723b7897509e616d3ee/tests/e2e/normalize.go#L354-L356

Reply:
ah man. thats correct. thanks .

Reply:
done.